src/main/java/org/terasology/anatomy/component/AnatomyPartTag.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.anatomy.component;
+
+import com.google.common.collect.Lists;
+import org.terasology.network.Replicate;
+import org.terasology.reflection.MappedContainer;
+
+import java.util.List;
+
+@MappedContainer
+public class AnatomyPartTag {
+    /**
+     * Id of the anatomy part. Useful for multi-word names.
+     */
+    public String id = "";
+
+    /**
+     * Name of the anatomy part.
+     */
+    @Replicate
+    public String name = "";
+
+    @Replicate
+    public List<String> effects = Lists.newArrayList();
```

Comment:
The AnatomyX effects are stored here in a list. This is essentially duplicate data of all the BrokenBoneEffect (and other effects to come).
I used Events to send the status info from the AnatomyX systems back to the base system. This duplication had to be done because all the events can't be received simultaneously (they are added when they are registered in their respective systems), so the temporary list had to be stored somewhere. Might as well store it in the component. This will allow the UI to directly look up the list of effects from the component as well.

Modifiable events only support integer values or we could have used them such that each system could add to the list when asked for.

Any alternative suggestions are appreciated.